### PR TITLE
Update acl && aws docs && fixed http demo error.

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

**Review by JaylinYu — APPROVED** [review]

File: docs/en_US/access-control/introduction.md [hunk 1]
```diff
@@ -7,60 +7,27 @@ Authentication and authorization are two fundamental security measures required
...
-```
-
-where, 
-
-- `no_match` determines the default action for a publish/subscribe request if none of the configured authorizers found any authorization rules.
-- `deny_action` determines what to do if a request was rejected according to the checks
--  `cache` is an optional value with caching settings.
-- `sources`  is an optional list that configures the chain of authorizers;
\ No newline at end of file
+Optional value with caching settings.
```
Comment (LenaLenaPan) [inline docs/en_US/access-control/introduction.md:29]:
```suggestion
Optional value with caching settings:
```

File: docs/en_US/access-control/introduction.md [hunk 2]
```diff
@@ -7,60 +7,27 @@ Authentication and authorization are two fundamental security measures required
...
+- `cache.max_size` — optional integer value, default is 32. Specifies the maximum number of elements in the cache. Older records are evicted from the cache when the specified number is exceeded.
```
Comment (LenaLenaPan) [inline docs/en_US/access-control/introduction.md:30]:
```suggestion
- `cache.max_size`: an optional integer value, default is 32. Specifies the maximum number of elements in the cache. When this limit is exceeded, the oldest records will be removed from the cache.
```

File: docs/en_US/access-control/introduction.md [hunk 3]
```diff
@@ -7,60 +7,27 @@ Authentication and authorization are two fundamental security measures required
...
+- `cache.ttl` — optional duration value, default is `1m`. Specifies how long cached values are kept in the cache.
```
Comment (LenaLenaPan) [inline docs/en_US/access-control/introduction.md:31]:
```suggestion
- `cache.ttl`: an optional duration value, default is `1m`. Specifies how long cached values are kept in the cache.
```

**Review by Hermann0222 — APPROVED** [review]

File: docs/zh_CN/access-control/introduction.md [hunk 4]
```diff
@@ -9,61 +9,25 @@
...
- 其中，
-
-- `no_match`：如当前客户端操作无法匹配到任何规则，将基于此规则决定允许或拒绝操作。
-- `deny_action`：如当前客户端的操作被拒绝，后续应执行的操作。
-- `cache`：授权缓存的相关配置。
-- `sources`（可选）：带顺序的数组，用于配置授权检查器的数据源。<!--带顺序吗？-->
\ No newline at end of file
+ACL 缓存的配置。
+- `cache.max_size` — 默认值 `32`。此配置规定每个客户端允许缓存的 ACL 规则数量。当超过上限时，老的记录将会被删掉。
+- `cache.ttl` — 默认 `1m`（一分钟）。 该配置规定 ACL 规则缓存有效时间。
```
Comment (LenaLenaPan) [inline docs/zh_CN/access-control/introduction.md:33]:
```suggestion
- `cache.ttl` — 默认 `1m`（一分钟）。 该配置规定 ACL 规则缓存有效时间。
- `password` — 指定密码文件路径，具体参考[登录认证](./login.md)。
- `acl` — 指定访问规则文件路径，具体参考[访问控制列表](./acl.md)。
```

File: docs/zh_CN/access-control/login.md [hunk 5]
```diff
@@ -9,40 +9,19 @@
 | `username` | 字符串 | 帐号 |
 | `password` | 字符串 | 密码 |
 
+请以以下格式将用户名和密码写入 `nanomq_pwd.conf` 文件：
+```shell
+username:password
+```
+并在 `nanomq.conf` 中 `include`，可参照 [访问控制介绍](./introduction.md)。
```
Comment (LenaLenaPan) [inline docs/zh_CN/access-control/login.md:16]:
```suggestion
并在 `nanomq.conf` 中 `include`，可参照 [访问控制介绍](./introduction.md)。
示例：
`password = {include "/etc/nanomq_pwd.conf"}`
```

File: docs/zh_CN/access-control/acl.md [hunk 6]
```diff
@@ -6,55 +6,28 @@
...
+| topics   | Array[String]  | 否   | 主题或主题过滤器数组                                         |
 | username | String         | 否   | 用户名若输入值为 "`#`"，表示所有用户                         |
 | clientid | String         | 否   | 客户端 ID 若输入值为 "`#`"，表示所有客户端                   |
 | and      | Array[Map]     | 否   | 与操作                                                       |
 | or       | Array[Map]     | 否   | 或操作                                                       |
 
 ## 配置实例
 
-在以下配置文件中，我们为不同的用户和主题定义了不同的访问规则，实现了灵活的授权检查机制。
+在以下配置文件中，我们为不同的用户和主题定义了不同的访问规则，实现了灵活的授权检查机制。请以正确的格式将规则写入 `nanomq_acl.conf` 文件，并在 `nanomq.conf` 中 `include`，可参照 [访问控制介绍](./introduction.md)。
```
Comment (LenaLenaPan) [inline docs/zh_CN/access-control/acl.md:17]:
```suggestion
在以下配置文件中，我们为不同的用户和主题定义了不同的访问规则，实现了灵活的授权检查机制。请以正确的格式将规则写入 `nanomq_acl.conf` 文件，并在 `nanomq.conf` 中 `include`，可参照 [访问控制介绍](./introduction.md)。
示例：
`acl = {include "/etc/nanomq_acl.conf"}`
```

Comment (lee-emqx) [reply docs/zh_CN/access-control/acl.md:17]: Thanks for your sugession, I will modify it later.

**Pull request merged**